bin/check

```diff
@@ -18,6 +18,8 @@ else
 
     RESOURCE="$RESOURCE_TYPE/$RESOURCE_NAME"
 
+    echo "$RESOURCE"
```

Comment:
it's cool, i think you can leave it in, just make sure that stdout is redirected (it may not since you're in a subshell, you'd have to check) and maybe add a message?